src/ViewHelpers/RenderViewHelper.php

```diff
@@ -121,7 +121,20 @@ public static function renderStatic(array $arguments, \Closure $renderChildrenCl
         $delegate = $arguments['delegate'];
         /** @var RenderableInterface $renderable */
         $renderable = $arguments['renderable'];
+
+        // Prepare a delegate variable provider that will be possible to extract after rendering the child closure.
+        // Any variable defined therein gets used as argument and overrides any argument of the same name.
+        // Note: not using late static binding here is a conscious decision: if late static binding had been used
```

Comment:
Just noticed that this comment is no longer correct, since removing the VH scope argument from the methods it relates to.